src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,16 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				if (ev.changedTouches === undefined) {
+					this._trigger('slideStart');
+					return;
+				}
+
+				var touch = ev.changedTouches[0];
+				this.touchX = touch.pageX;
+				this.touchY = touch.pageY;
```

Comment:
if you remove the function call from `line 1274` and call `this._mousedown(ev)` here, does that also work?